pytorch_toolkit/object_detection/configs/face-detection-0100.py

```diff
@@ -0,0 +1,144 @@
+import os
+
+# model settings
+input_size = 256
+width_mult = 1.0
+model = dict(
+    type='SingleStageDetector',
+    backbone=dict(
+        type='mobilenetv2_w1',
+        out_indices=(4, 5),
+        frozen_stages=-1,
+        norm_eval=False,
+        pretrained=True,
+    ),
+    neck=None,
+    bbox_head=dict(
+        type='SSDHead',
+        input_size=input_size,
+        in_channels=(int(width_mult*96), int(width_mult*320)),
+        num_classes=2,
+        anchor_strides=(16, 32),
+        anchor_widths=( [  9.4 * 0.85333333333,  25.1 * 0.85333333333,  14.7 * 0.85333333333,  34.7 * 0.85333333333],
+                        [143.0 * 0.85333333333,  77.4 * 0.85333333333, 128.8 * 0.85333333333,  51.1 * 0.85333333333, 75.6 * 0.85333333333]),
+        anchor_heights=([ 15.0 * 0.85333333333,  39.6 * 0.85333333333,  25.5 * 0.85333333333,  63.2 * 0.85333333333],
+                        [227.5 * 0.85333333333, 162.9 * 0.85333333333, 124.5 * 0.85333333333, 105.1 * 0.85333333333, 72.6 * 0.85333333333]),
+        target_means=(.0, .0, .0, .0),
+        target_stds=(0.1, 0.1, 0.2, 0.2),
+        depthwise_heads=True,
+        depthwise_heads_activations='relu',
+        loss_balancing=True))
```

Comment:
Offtop: does it really work?

Reply:
to be honest, I did not play with it. I just trusted you.